apps/namadillo/src/hooks/useAssetAmount.tsx

```diff
@@ -14,7 +14,8 @@ type useAmountTransferProps = {
 type UseAmountTransferOutput = {
   isLoading: boolean;
   balance: BigNumber | undefined;
-  availableAssets: Asset[];
+  availableAssets?: Asset[];
```

Comment:
I left `availableAssets` as optional because it will be easier to identify if their content has been loaded or not. But you're right, the way it was implemented wouldn't ever let this be undefined. Fixed!